src/lib/types.ts

```
@@ -1079,7 +1080,7 @@ export interface NavbarProps extends Omit<HTMLAttributes<HTMLDivElement>, "child
   breakpoint?: NavbarBreakpoint;
 }
 
-export interface NavBrandProps extends HTMLAnchorAttributes {}
+export interface NavBrandProps extends HTMLAnchorAttributes { }
```

ESLint

[error] An interface declaring no members is equivalent to its supertype.

(@typescript-eslint/no-empty-object-type)